doc/tutorials/dicom_json.rst

```diff
@@ -94,3 +94,14 @@ the stored ``BulkDataURI``:
   >>>     "00091002": {"vr": "OB", "BulkDataURI": "https://my.wado.org/123"}
   >>> }
   >>> ds = Dataset.from_json(json_data, bulk_data_uri_handler=bulk_data_reader)
+
+or, if you need to also know the tag and the vr, in addition to the stored
+``BulkDataURI``:
+
+  >>> def bulk_data_reader(tag, vr, bulk_data_uri):
+  >>>     return data_retrieved_via_uri(bulk_data_uri)
+  >>>
```

Comment:
Ideally should show the function passing using `tag` and `vr` here, otherwise no difference from previous example.

Reply:
Would you like me to do something with this, or is it OK?

Reply:
I think we might as well change it while we are here - the existing example seems odd to me anyway, just to refer to some other function.  But simplest fix is maybe just `return data_retrieved_for_tag_and_vr(tag, vr, bulk_data_uri)` or similar.

Reply:
Done.